src/config.ts

```diff
@@ -20,6 +20,8 @@ export interface BatchResourceConfig {
     isBatchResource: true;
     batchKey: string;
     newKey: string;
+    secondaryBatchKey: string;
+    secondaryNewKey: string;
```

Comment:
I believe [`schema.json`](https://github.com/Yelp/dataloader-codegen/blob/master/schema.json#L93) in the root of the repo needs to be updated for these new config options as well.